packages/fela/README.md

```diff
@@ -1,237 +1,237 @@
-<h1><img alt="Fela" src="docs/res/logo.png" width="203"/></h1>
-

```

Comment:
For some reason the whole file is marked as changed. Was it actually edited? I failed to find the changes

Reply:
This file had CRLF line endings I accidentally touched it and now they are LF
Will try to revert these changes

Reply:
Reverted 👍

Reply:
This file is auto-generated/copied on release, so it doesn't matter anyways :P